boards/nordic/nrf52dk_base/src/lib.rs

```diff
@@ -12,10 +11,11 @@ use kernel::capabilities;
 use kernel::common::dynamic_deferred_call::{DynamicDeferredCall, DynamicDeferredCallClientState};
 use kernel::component::Component;
 use kernel::hil;
+use kernel::hil::gpio::{Configure, FloatingState};
 use nrf52::gpio::Pin;
 use nrf52::rtc::Rtc;
-use nrf52::uicr::Regulator0Output;
-
+use nrf52::uicr::Regulator0Output; //TODO is this temporary?
```

Comment:
not sure why this is changed in this PR....can I change it back?

Reply:
yeah definitely, for this line and the line below it